bundles/pom.xml

```diff
@@ -43,7 +43,34 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-scr-plugin</artifactId>
+        <version>1.24.0</version>
+        <configuration>
+          <supportedProjectTypes>
+            <supportedProjectType>eclipse-plugin</supportedProjectType>
+          </supportedProjectTypes>
+        </configuration>
+        <executions>
+          <execution>
+            <id>generate-scr-scrdescriptor</id>
+            <goals>
+              <goal>scr</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>      
     </plugins>
   </build>
 
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.felix</groupId>
+      <artifactId>org.apache.felix.scr.ds-annotations</artifactId>
```

Comment:
PS: Just for clarification - this dependency brings the DS annotation processor (in this case for the standard OSGi annotations) into the build. It does NOT contain the DS Annotations themselves. They get resolved via the usual target platform by having the optional package dependency to `org.osgi.service.component.annotations`.